tests/Set-DbaLogin.Tests.ps1

```diff
@@ -5,11 +5,10 @@ Write-Host -Object "Running $PSCommandpath" -ForegroundColor Cyan
 Describe "$CommandName Unit Tests" -Tag 'UnitTests' {
     Context "Validate parameters" {
         $command = Get-Command $CommandName
-        [object[]]$params = $command.Parameters.Keys
-        $knownParameters = 'SqlInstance', 'SqlCredential', 'Login', 'SecurePassword', 'Unlock', 'MustChange', 'NewName', 'Disable', 'Enable', 'DenyLogin', 'GrantLogin', 'AddRole', 'RemoveRole', 'EnableException', 'InputObject'
-        $paramCount = $knownParameters.Count
-        It "Contains our specific parameters" {
-            ((Compare-Object -ReferenceObject $knownParameters -DifferenceObject $params -IncludeEqual | Where-Object SideIndicator -eq "==").Count) | Should Be $paramCount
+        [object[]]$params = (Get-ChildItem function:\Set-DbaLogin).Parameters.Keys
```

Comment:
With this change you can just remove line 7 altogether, it is redudant.

Reply:
when i did, a bunch of stuff failed so i added it back then it passed